src/util/debug_module.js

```diff
@@ -539,6 +539,9 @@ DebugLogger.prototype.log2_withbt = log_bt_builder(2);
 DebugLogger.prototype.log3_withbt = log_bt_builder(3);
 DebugLogger.prototype.log4_withbt = log_bt_builder(4);
 
+if (Number(process.env.NOOBAA_LOG_LEVEL)) {
```

Comment:
I just noticed another thing - why did you choose this location in the file, where this statement is breaking the flow of defining DebugLogger class methods? I think I would push this all the way down.